include/onnxruntime/core/framework/ortdevice.h

```diff
@@ -17,6 +17,7 @@ struct OrtDevice {
   static const DeviceType GPU = 1;  // Nvidia or AMD
```

Comment:
Would it be clearer (but possibly a much large search and replace, and so I understand the hesitation) to rename this to CUDA?

Reply:
Yeah, trying not to boil the ocean for this change. Your comment is well taken though and something that needs cleanup.